src/corner-dialog/src/CornerDialog.js

```diff
@@ -243,7 +242,7 @@ export default class CornerDialog extends PureComponent {
               data-state={state}
               padding={32}
               position="fixed"
-              {...position}
+              {...AbsolutePosition[position]}
```

Comment:
Nit: prefer camelCase for imports that are not components/classes.

Reply:
We also should validate that it returns something – e.g. if it's not one of the 4 supported corners we should default to `positions[Position.BOTTOM_RIGHT]`

Reply:
We have this on default props. And with `oneOf[...]`

Reply:
Right, but you could still pass invalid values – and just get warned.

We need it to always pick a valid value – so if there's a value passed that is incorrect we should either `throw` an error or gracefully fall back to the default.